app/src/main/res/layout/beneficiary_spinner_layout.xml

```diff
@@ -12,13 +12,13 @@
         android:layout_height="wrap_content"
         android:layout_width="match_parent"
         android:textAppearance="?android:textAppearanceMedium"
-        tools:text="1"/>
+        tools:text="@string/one"/>
```

Comment:
It wasn't needed even though it causes no harm :) You are only supposed to shift "android:text" not "tools:text" :)